06-team-data-github-api/src/App.js

```diff
@@ -0,0 +1,69 @@
+import ReactDOM from 'react-dom/client';
+import React, { useState, useEffect } from 'react';
+
+import './index.css';
+
+import CardComponent from './component/CardComponent';
+import NavbarComponent from './component/NavbarComponent';
+import gitUserNames from './util/Constant';
+
+const CardHolder = ({ filteredTeam }) => {
+  if (filteredTeam.length > 0) {
+    return filteredTeam.map((user) => {
+      if (user.login) {
+        return (
+          <CardComponent
+            user={user}
+            key={user.id}
+          />
+        );
+      }
+    });
+  }
+};
+
+const AppLayoutComponent = () => {
+  const [teamsList, setTeamsList] = useState([]);
+  const [filteredList, setFilteredList] = useState([]);
+  const [isMemberPresent, setIsMemberPresent] = useState(true);
+  useEffect(() => {
+    fetchData();
+  }, []);
+  async function fetchData() {
```

Comment:
```suggestion
  const [teamsList, setTeamsList] = useState([]);
  const [filteredList, setFilteredList] = useState([]);
  const [isMemberPresent, setIsMemberPresent] = useState(true);
  
  useEffect(() => {
    fetchData();
  }, []);
  
  async function fetchData() {
```
Space out your declarations and functions for better readability